User/UserRepository.php

```diff
@@ -20,4 +20,38 @@ public function findByUsername($username);
      * @return bool
      */
     public function supportsClass($class);
+
+    /**
+     * Add User to the repository.
+     *
+     * @param User $user
+     * @param bool $save
+     * @return void
+     */
+    public function add(User $user, $save = true);
+
+    /**
+     * Flush the repository.
```

Comment:
die comment wijst al op doctrine. Ik zou hier gewoon "saves the given user" ofzo zetten. Of dat daarvoor moet geflushed worden of iets heel anders doet er niet echt toe.